students/BrianB/lesson05/except_exercise.py

```diff
@@ -42,13 +50,24 @@
 
 langs = ['java', 'c', 'python']
 
+# <<<<<<< master
+try:
+    more_joke = more_fun(langs[0])
+except IndexError:
+    pass
+
+# =======
 more_joke = more_fun(langs[0])
+# >>>>>>> master
 
 try:
     not_more_joke = more_fun(langs[1])
 except SyntaxError:
     print('Run Away!')
 else:
+# <<<<<<< master
+# =======
     more_fun(langs[0])
```

Comment:
Same thing here, I think you meant this previously but it's still here outside a try block. I think you meant to remove this as well.